pelican/log.py

```diff
@@ -92,10 +92,15 @@ class LimitFilter(logging.Filter):
     """
 
     _ignore = set()
+    _raised_messages = set()
     _threshold = 5
     _group_count = defaultdict(int)
 
     def filter(self, record):
+        # don't filter log messages when "debug" is enabled
+        logger_level = logging.getLogger().getEffectiveLevel()
+        if logger_level == logging.DEBUG:
+            return True
```

Comment:
> Also I don't understand the intention behind this change.
> I see myself only using this feature in debug mode to reduce the logging flood.

I would actually want to use this filtering in production, not debug mode. There is at least one warning type that I consider wrong (the one about empty `alt` attributes for images, as there are good reasons to have empty `alt`s), and I would like to have a way to disable it. It is a string-interpolated warning, hence the current LOG_FILTER does not work on them, but this PR would solve that problem.